test/controllers/graphql_controller_11_test.rb

```diff
@@ -397,4 +397,30 @@ def teardown
     assert_response :success
     assert_match /ActiveRecord::RecordNotFound/, @response.body
   end
+
+  test "should query tipline_requests that never received articles" do
```

Comment:
Please rename this test to reflect the two things you're doing (not only querying requests, but also sending explainers).

Reply:
Done